packages/components/inputs/search-select-input/src/search-select-input-option.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import { components } from 'react-select';
+import { NO_VALUE_FALLBACK, SELECT_DROPDOWN_OPTION_TYPES } from './constants';
+
+export const FullDetailedSearchSelectInputOption = (props) => {
+  const { data } = props;
+  return (
+    <components.Option {...props}>
+      <Spacings.Stack scale="xs">
+        <Text.Detail isBold={true}>
+          {data.label || NO_VALUE_FALLBACK}
```

Comment:
Should we allow setting the fallback value as a prop? 
It might be a bit restrictive to have it fixed to '----', which can easily change in the future or depend on the context.

Reply:
We can `fallback` prop and if given that will get precedence